blockchain/transact.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+# Create JSON files for permissions
```

Comment:
i think this would work differently. DRAFT ONLY to show intention. Not sure the actual code.

```bash
# get the password securly
passwd=$(readlink -f passed)

# import the key
cleos import key $passwd

# create an alias so that it connects to pangea mainnet
alias cleos='cleos -u https://pangea-mainnet-api.com'

# open terminal. user can then do cleos commands easily with key and connected to mainnet
/bin/bash
```

this would then open a terminal in the container, with the Key import. Then we can give them transactions to send and it will send on their behalf. This is a flexible approach so that they can execute whatever transactions they want to try.